### PR TITLE
Chore: Improve image build time

### PR DESCRIPTION
**Pull request by tabossert**
- Builds from pre built custom image
- Has some fixes from other branches (SSL, Makefile changes)

https://unstructured-ai.atlassian.net/browse/CORE-863?search_id=6f32eee6-c566-4a04-ba27-d14e802f55b6

### CODE REVIEW

**Comment by tabossert** [comment]
I still need to get the M-series builds going, but this is a start.  We can use the same base image for unstructured and other codebases

**Review by ryannikolaidis — APPROVED** [review]
LGTM!

```

File: README.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ When elements are extracted from PDFs or images, it may be useful to get their b
 	`pyenv  virtualenv 3.8.15 document-processing` <br />
 	`pyenv activate document-processing`
 
+  - To install the `detectron2` model, which
+  `unstructured` uses for layout detection:
+    - make install-detectron2
```
Comment (cragwolfe) [inline README.md:76]: nit: think this block fits more naturally combined with " Run make install" line a few lines below. I.e., "if image and high resolution pdf extraction is required, also run `make install-detectron2`"

... and actually ...

the default image layout model probably won't be detectron2, so i'd name it something more abstract to begin with.

**Review by ryannikolaidis — CHANGES_REQUESTED** [review]
Actually maybe worth holding for M series changes? That way we don't regress until then?

**Comment by tabossert** [comment]
Sounds good, I will get M series working tomorrow morning

**Comment by ryannikolaidis** [comment]
testing arm build now

**Review by ryannikolaidis — APPROVED** [review]
LGTM! 🥳

ran:

```
>> make docker-build
>> docker inspect pipeline-family-general-dev:latest | grep Arch
        "Architecture": "arm64",

>> make docker-start-api 
docker run -p 8000:8000 --mount type=bind,source=/Users/ryannikolaidis/Development/unstructured/unstructured-api,target=/home/notebook-user/local -t --rm pipeline-family-general-dev:latest --log-config logger_config.yaml --host 0.0.0.0 --port 8000
2023-05-25 23:34:45,094 uvicorn.error INFO Started server process [1]
2023-05-25 23:34:45,094 uvicorn.error INFO Waiting for application startup.
2023-05-25 23:34:45,094 uvicorn.error INFO Application startup complete.
2023-05-25 23:34:45,096 uvicorn.error INFO Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
[ran request]
2023-05-25 23:34:58,501 172.17.0.1:55024 POST /general/v0/general HTTP/1.1 - 200 OK
```

**Pull request merged**